docs/sources/whatsnew/whats-new-in-v10-2.md

```diff
@@ -186,4 +186,26 @@ With the new **Format string** transformation, you can manipulate string fields
 
 Learn more in the [Format string documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/query-transform-data/transform-data/#format-string).
 
+### Detect Unusable Transformations
+
+<!-- Kyle Cunningham -->
+
+_Available behind a feature flag in Grafana Open Source and Enterprise_
```

Comment:
We should talk about this in terms of a release stage. Does this align with [experimental, private preview, or public preview](https://grafana.com/docs/release-life-cycle/)?